packages/scandipwa/src/component/Field/Field.container.js

```diff
@@ -79,6 +79,10 @@ export class FieldContainer extends PureComponent {
 
     fieldRef = createRef();
 
+    componentDidMount() {
+        this.validate();
+    }
```

Comment:
We have some form fields, that shouldn't be validated on load, so this solution won't work.

Instead change in **GroupedProductsItem.component.js**:
from: `validateOn={ ['onChange', 'onload'] }`
to: `validateOn={ ['onChange', 'onLoad'] }`
_(there was typo mistake for "onLoad" (capital L) event, thus it wasn't executing on load)_